pkg/provisioner/provisioner.go

```diff
@@ -71,10 +71,8 @@ func New(log *logger.FunLogger, keyPath, userName, hostUrl string) (*Provisioner
 }
 
 func (p *Provisioner) Run(env v1alpha1.Environment) error {
-	graph, err := buildDependencyGraph(env)
-	if err != nil {
-		return fmt.Errorf("failed to build dependency graph: %v", err)
-	}
+	solver := NewDependencySolver(env)
+	solver.ResolveDependencies()
```

Comment:
Qustion: Why does `a resolver` not return the dependencies?

Reply:
addressed